### DISCUSSION TITLE
크롬 주소창에 붙여넣기했는데 반응이 없습니다

### DISCUSSION THREAD

**Discussion by Min0317** [kimytsc/covid-rest-vaccine-macro#21] [Q&A]
소스 편집해서 붙여넣었는데
그냥 바로 꺼져버리네요,뭐가 문제일까요?

**Comment by kimytsc** [comment]
> 소스 편집해서 붙여넣었는데
> 그냥 바로 꺼져버리네요,뭐가 문제일까요?

조금전 테스트해보니 정상적으로 동작하는데, 혹시 앞에 `javascript:`를 직접 적으신 후에 붙여넣으셨나요?

주소창에 `javascript:` 로 시작하는 소스를 복붙하면 `javascript:`가 없어진 상태로 주소창에 붙여넣어집니다.
ex) `javascript:alert("테스트");` 를 복사해서 붙여넣으면 `alert("테스트");`만 주소창에 붙여넣어짐

**Reply by Min0317** [reply]
네 자바스크립트 친 상태로 붙여넣었어요

**Reply by kimytsc** [reply]
혹시 여전히 동일한 현상을 보이고 있으신가요?

디버깅을 위해 붙여넣으신 코드 전달 가능할까요?
